src/playerHandler.ts

```diff
@@ -0,0 +1,39 @@
+import { AttackHandler } from './attackHandler';
+import { Collidable } from './collidable';
+import { Enemy } from './enemy';
+import { MovementHandler } from './movementHandler';
+import { PhysObject } from './physObject';
+import { Player } from './player';
+
+export class PlayerHandler {
```

Comment:
Same thing about simply exporting the functions here rather than having a class.

Reply:
done